app/components/atoms/number-heading/number-heading.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import {ComponentMeta, ComponentStory} from '@storybook/react';
+
+import {NumberHeading} from './number-heading.component';
+
+export default {
+  title: 'Atoms/NumberHeading',
+  component: NumberHeading,
+} as ComponentMeta<typeof NumberHeading>;
+
+const Template: ComponentStory<typeof NumberHeading> = (args) => (
+  <NumberHeading {...args} />
```

Comment:
I made some changes just based on the `children` name:

```typescript
import {ComponentMeta, ComponentStory} from '@storybook/react';

import {NumberHeading} from './number-heading.component';

export default {
  title: 'Atoms/NumberHeading',
  component: NumberHeading,
} as ComponentMeta<typeof NumberHeading>;

const Template: ComponentStory<typeof NumberHeading> = (props) => (
  <NumberHeading>
    {props.children}
  </NumberHeading>
);

export const Number = Template.bind({});
Number.args = {
  children: "1.",
};
```

So I am more concerned on the usage of this component now:


```typescript
<NumberHeading>1.</NumberHeading>
```

For the name it is a little bit long... also the name is misleading as I can use


```typescript
<NumberHeading>This is my number heading</NumberHeading>
```

And it would look like this:
![image](https://user-images.githubusercontent.com/339040/229655956-bd00bdd3-4d7e-4ea7-8cce-b07e330ac57e.png)

So not much of a number heading only right?